app/res/xml/preferences_developer.xml

```diff
@@ -45,6 +43,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-session-saving"
         android:title="Enable Session Saving"/>
+    <EditTextPreference
```

Comment:
@phillipm Pinging on this question, and also the build is still failing, but when I go to click on 'Details', it says the build is not found. Any idea what's up?